easy-contact/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
`.gitignore` should be placed in the root directory

Reply:
moved the files to the root directory